src/api/img.ts

```diff
@@ -32,24 +33,26 @@ export default ({ config, db }) =>
       action = req.query.action
     } else {
       let urlParts = req.url.split('/');
-      width = parseInt(urlParts[1]);
-      height = parseInt(urlParts[2]);
-      action = urlParts[3];
-      imgUrl = `${config[config.platform].imgUrl}/${urlParts.slice(4).join('/')}`; // full original image url
-  
-      if (urlParts.length < 4) {
+
+      if (urlParts.length < 5) {
```

Comment:
It must be backward compatible - so it might be a situation when the user doesn't provide the `type` and we have some deafult assumption in here. If there is no `type` we also need to have the `product` by default in the `imgUrl` - please keep this one in mind as well